integration-tests/reactive-messaging-kafka/src/main/java/io/quarkus/it/kafka/PeopleManager.java

```diff
@@ -0,0 +1,27 @@
+package io.quarkus.it.kafka;
+
+import java.util.List;
+import java.util.concurrent.CopyOnWriteArrayList;
+
+import org.eclipse.microprofile.reactive.messaging.Incoming;
+import org.jboss.logging.Logger;
+
+import io.vertx.core.json.JsonObject;
+
+public class PeopleManager {
+
+    private final Logger log = Logger.getLogger(PeopleManager.class);
+
+    private final List<Person> list = new CopyOnWriteArrayList<>();
+
+    @Incoming("people-in")
+    public void consume(String json) {
+        Person person = new JsonObject(json).mapTo(Person.class);
+        list.add(person);
+    }
```

Comment:
```suggestion
    public void consume(Person json) {
        list.add(person);
    }
```

Good PR. Should we use the value deserializer since we have it in config?

Reply:
Yes, you are right.